### ISSUE TITLE
Weird artifact in Tabs component

### ISSUE THREAD

**Issue by sharpcodepro** [IBM/carbon-components-svelte#241]
In the Tabs component there is a weird element in the beginning (marked in red on the screenshot).

<img width="950" alt="Screen Shot 2020-09-06 at 14 18 20" src="https://user-images.githubusercontent.com/34513877/92325548-12b11f80-f04c-11ea-8d5c-c55dd956597d.png">

In the component HTML, I commented out relevant part (shown below) and it is gone. What is the purpose of this dropdown? I can't seem to find it in CDS documentation, neither in implementation for other frameworks (Vue in this case).

```
<div
    role="listbox"
    tabindex="0"
    class:bx--tabs-trigger={true}
    aria-label={$$props['aria-label'] || 'listbox'}
    on:click={() => {
      dropdownHidden = !dropdownHidden;
    }}
    on:keypress
    on:keypress={() => {
      dropdownHidden = !dropdownHidden;
    }}>
    <a
      tabindex="-1"
      class:bx--tabs-trigger-text={true}
      href={triggerHref}
      on:click
      on:click={() => {
        dropdownHidden = !dropdownHidden;
      }}>
      {#if currentTab}{currentTab.label}{/if}
    </a>
    <ChevronDownGlyph aria-hidden="true" title={iconDescription} />
  </div>
```

**Comment by metonym** [comment]
@sharpcodepro thanks for raising the issue.

This is caused by a breaking change in `carbon-components` v10.19.x. The mobile behavior for tabbed content is completely changed.

What you're seeing is the mobile menu that is now unstyled.

For now, the best recommendation I have is to downgrade to `carbon-components` v10.18.

**Comment by metonym** [comment]
Due to the severity of this issue, In #243, I'm exploring shipping pre-compiled CSS styles with this library that cherry pick component styles from different versions of `carbon-components`.

**Comment by sharpcodepro** [comment]
I think current arrangement is quite flexible when components do not include styles. For example, I wanted to apply the g90 theme and it was not immediately apparent how to do so. Then I have removed https://unpkg.com/carbon-components/css/carbon-components.min.css altogether. Rather, I have imported @import 'carbon-components/scss/globals/scss/styles.scss' into an scss file and got all flexibility to customize the styles. If pre-compiled CSS styles are included, this flexibility will be gone or no?
